server/actions/Event.ts

```diff
@@ -41,6 +41,24 @@ export const getEvent = async function (id: string) {
     return event;
 };
 
+/**
+ * @param id EventId string to identify an event in our database.
+ * @returns The number of volunteers attending an given
+ */
+export const getAttendedCount = async function (id: string) {
+    await mongoDB();
+    if (!id || id == "") {
+        throw new APIError(400, "Invalid id");
+    }
+
+    const event = await EventSchema.findById(id);
```

Comment:
Since we're only interested in the length of `attendedVols`, can you use projection to return only the `attendedVols` field from the query? https://docs.mongodb.com/manual/tutorial/project-fields-from-query-results/

Reply:
Should be changed now, thanks for letting me know!